### ISSUE TITLE
deleting a character or inserted text is not read by screen readers in terminal

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#44080]
Does this issue occur when all extensions are disabled?: Yes
• VS Code Version: 1.88.1
• OS Version: Windows_NT x64 10.0.22631
Steps to Reproduce:
1. Turn on NVDA screen reader.
2. Open VS Code and press Ctrl+` to open the terminal.
3. Type something in the terminal. If you delete a character or use Insert+Up Arrow, the screen reader does not announce the changes.
4. This issue does not occur in PowerShell or CMD.
Expected: Screen readers should announce all text changes in the terminal.
Actual: No announcements when text is deleted or read using Insert+Up Arrow.
